ide/build.cjs

```diff
@@ -25,7 +25,7 @@ let common = {
 };
 
 let extension = estrella.build({
-  entryPoints: ["src/extension.ts"],
+  entryPoints: ["src/extension.ts", "src/tests/slice/util/slice_helpers.ts"],
```

Comment:
I don't think we want to have `slice_helpers.ts` be an entryPoint in the same build. This would include `slice_helpers.js` in the distribution, but it's only used for development.

I would either conditionally add it as an entry point, or add another `estrella.build` call with a different `outdir` that isn't `dist`.